[2023-10-01 09:51:40,596] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: postgres_v1.create_postgres_connection 2021-08-05T00:00:00+00:00 [queued]>
[2023-10-01 09:51:40,745] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: postgres_v1.create_postgres_connection 2021-08-05T00:00:00+00:00 [queued]>
[2023-10-01 09:51:40,763] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-10-01 09:51:40,767] {taskinstance.py:1043} INFO - Starting attempt 5 of 6
[2023-10-01 09:51:40,771] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-10-01 09:51:40,883] {taskinstance.py:1063} INFO - Executing <Task(PostgresOperator): create_postgres_connection> on 2021-08-05T00:00:00+00:00
[2023-10-01 09:51:41,161] {standard_task_runner.py:52} INFO - Started process 42606 to run task
[2023-10-01 09:51:41,441] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'postgres_v1', 'create_postgres_connection', '2021-08-05T00:00:00+00:00', '--job-id', '2593', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/1_postgresql.py', '--cfg-path', '/tmp/tmpx83gc95s', '--error-file', '/tmp/tmpnfxc0edf']
[2023-10-01 09:51:41,765] {standard_task_runner.py:77} INFO - Job 2593: Subtask create_postgres_connection
[2023-10-01 09:51:43,509] {logging_mixin.py:104} INFO - Running <TaskInstance: postgres_v1.create_postgres_connection 2021-08-05T00:00:00+00:00 [running]> on host d4ed655215d1
[2023-10-01 09:51:44,579] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=svastikkka
AIRFLOW_CTX_DAG_ID=postgres_v1
AIRFLOW_CTX_TASK_ID=create_postgres_connection
AIRFLOW_CTX_EXECUTION_DATE=2021-08-05T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-05T00:00:00+00:00
[2023-10-01 09:51:44,586] {postgres.py:69} INFO - Executing: create table if not exists dags_run(
        dt  date,
        dag_id character varying,
        primary key (dt,dag_id)
        )
[2023-10-01 09:51:44,705] {base.py:74} INFO - Using connection to: id: posgres_testing. Host: host.docker.internal, Port: 5432, Schema: testing, Login: airflow, Password: XXXXXXXX, extra: None
[2023-10-01 09:51:44,760] {dbapi.py:180} INFO - Running statement: create table if not exists dags_run(
        dt  date,
        dag_id character varying,
        primary key (dt,dag_id)
        ), parameters: None
[2023-10-01 09:51:44,796] {dbapi.py:186} INFO - Rows affected: -1
[2023-10-01 09:51:44,808] {postgres.py:73} INFO - NOTICE:  relation "dags_run" already exists, skipping

[2023-10-01 09:51:44,958] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=postgres_v1, task_id=create_postgres_connection, execution_date=20210805T000000, start_date=20231001T095140, end_date=20231001T095144
[2023-10-01 09:51:46,244] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-01 09:51:49,548] {local_task_job.py:146} INFO - Task exited with return code 0
